sources/packages/backend/apps/queue-consumers/src/services/student-file/student-file.service.ts

```diff
@@ -94,4 +112,24 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
         );
     }
   }
+
+  /**
+   * Updates the infected file status and renames file to describe an error with the file.
+   * @param studentFile student infected file to be updated.
+   * @returns the update result.
+   */
+  private async updateInfectedFile(
+    studentFile: StudentFile,
+  ): Promise<UpdateResult> {
+    const fileName = path.parse(studentFile.fileName).name;
+    const fileExtension = path.parse(studentFile.fileName).ext;
```

Comment:
Please avoid calling the method twice. The result from `ath.parse(studentFile.fileName)` should be saved and have its properties accessed.